### DISCUSSION TITLE
请问AX86U merlin固件刷哪个版本可以用？

### DISCUSSION THREAD

**Discussion by Xunyu** [messense/aliyundrive-webdav#617] [Q&A]
请问AX86U merlin固件刷哪个版本可以用？

**Comment by tsxues** [comment]
【2022年10月11日 22:06:33】: ========================== step 1 ===============================
【2022年10月11日 22:06:33】: 开启插件离线安装！
【2022年10月11日 22:06:33】: 检测到你上传的离线安装包：aliyundrivewebdav-merlin-arm384-aarch64.tar.gz，安装包大小: 3.2M
【2022年10月11日 22:06:33】: 安装包md5sum校验值：48e4fe34a531a64f2a42b01f4414cbf2
【2022年10月11日 22:06:33】: 尝试解压离线安装包离线安装包...
【2022年10月11日 22:06:33】: 安装包解压成功！继续！
【2022年10月11日 22:06:33】: 当前jffs分区剩余：29496KB, 空间满足，继续安装！
【2022年10月11日 22:06:33】: 准备安装插件：aliyundrivewebdav
【2022年10月11日 22:06:33】: 运行安装脚本...
【2022年10月11日 22:06:33】: ========================== step 2 ===============================
【2022年10月11日 22:06:33】: 本插件适用于【koolshare merlin aarch64 / merlin_hnd】固件平台，你的平台不能安装！！！
【2022年10月11日 22:06:33】: ========================== step 3 ===============================
【2022年10月11日 22:06:33】: ks_tar_install.sh: 因为插件aliyundrivewebdav安装失败！退出离线安装！
【2022年10月11日 22:06:33】: 删除文件夹-1：/tmp/aliyundrivewebdav
【2022年10月11日 22:06:33】: 删除文件：/tmp/aliyundrivewebdav-merlin-arm384-aarch64.tar.gz
【2022年10月11日 22:06:33】: ============================ end ================================



也是AX86U merlin固件，刷的是aliyundrivewebdav-merlin-arm384-aarch64.tar，显示平台不能安装，不知道怎么处理了

**Reply by arekyao** [reply]
这个问题是因为代码对merlin的新的固件没有识别导致的，问题在install.sh中。
找到
if [ "$(uname -o|grep Merlin)" ] && [ -d "/koolshare" ];then
改成这样即可
if [ "$(uname -o|grep Koolcenter)" ] && [ -d "/koolshare" ];then
